modules/service/main.tf

```diff
@@ -814,6 +814,7 @@ module "container_definition" {
   # Container Definition
   command                = each.value.command
   cpu                    = each.value.cpu
+  credentialSpecs        = each.value.credentialSpecs
```

Comment:
this needs to be propagated to all instances throughout the module https://github.com/terraform-aws-modules/terraform-aws-ecs/blob/b8e633547ca4bc628af7db96b0044869c44b08e4/variables.tf#L448-L450